services/base/shared_libs/src/scanhub_libraries/models.py

```diff
@@ -229,12 +230,13 @@ class Config:
         # }
 
     workflow_id: Optional[UUID] = None  # Field("", description="ID of the workflow the task belongs to.")
+    name: str
     description: str
     type: TaskType
     args: dict[str, str]
     artifacts: dict[str, str]
     destinations: dict[str, str]
-    status: dict[TaskStatus, str]
+    status: TaskStatus
```

Comment:
There might be several task statuses and each status should or can have a description, which is why this should be a dictionary

Reply:
Ok, if you think that is necessary. I'll take that change back.

Reply:
I think this is what we agreed on (#90). We can also discuss it again, but I would not just change it. Perhaps we should stick more closely to issue-driven development in the future. I know it is more effort, but I think the refactoring is mostly done, and we could start tracking new features/improvements again using the issues.

Reply:
Ok, yes I'll try to link changes closer to issues. Changed back.